next.clayui.com/content/docs/css/components/forms/text-input-multi-select.md

```diff
@@ -0,0 +1,515 @@
+---
+title: "Multi Select"
+description: "Multi select is the field type that allows writing text to create “tags” that are represented in the shape of labels. This components counts with auto complete feature."
```

Comment:
This phrasing is a little confusing "This components counts with auto complete feature."

Reply:
Hm yeah, I'm going to remove that.